.github/release.yml

```
@@ -6,6 +6,13 @@ changelog:
       exclude:
         labels:
           - dependencies
-    - title: Dependencies 📦
+          - feature toggled
+          - technical changes only
+    - title: "Experimental features for testing 😎"
```

For this one, which really means "behind a feature toggle", I take inspiration from [JB](https://github.com/openfoodfoundation/openfoodnetwork/releases/tag/v4.4.2) and suggest something like:

```suggestion
    - title: "Features under construction 🚧"
```

---

> Features under construction 🚧

I like it. It fulfils the purpose better:

* Some instance managers want to know about updates here. This was first requested by Kirsten to know when she can test and what to tell customers. And for instances with the feature toggled on, these are user facing changes.
* Most instance managers should ignore this. Rachel pointed out that it's a nightmare when lesser involved instances start to use features without knowing the full impact and potential problems. So "under construction" conveys much better to stay clear unless you are a construction worker.